broadinstitute/single_cell_portal_core
# Updating to latest version of caniuse-lite (SCP-4043)

We have been getting deprecation warnings in our [build logs](https://github.com/broadinstitute/single_cell_portal_core/runs/4971075302?check_suite_focus=true#step:4:2117) ever since node-fetch was updated in a previous [PR](https://github.com/broadinstitute/single_cell_portal_core/pull/1326):

```
Browserslist: caniuse-lite is outdated. Please run:
  npx browserslist@latest --update-db
  Why you should do it regularly: https://github.com/browserslist/browserslist#browsers-data-updating
```

This update follows the [instructions](https://github.com/browserslist/browserslist#browsers-data-updating) from `browserslist` to update `caniuse-lite` to the latest version.  In addition, `yarn-deduplicate` was run to reduce the number of redundant versions of various packages are being loaded.

MANUAL TESTING
1. Pull this branch and run `./rails_local_setup.rb && source config/secrets/.source_env.bash`
2. Run `yarn install` and validate everything installs correctly
3. Run `yarn ui-test` and validate that all tests pass
4. Run `bin/bundle exec rake assets:clean && bin/bundle exec rake assets:precompile` and validate that the deprecation warning for `caniuse-lite` does not appear
